.github/workflows/ci.yml

```diff
@@ -51,8 +51,8 @@ jobs:
     - uses: actions/checkout@v2
     - run: which go && go version && which cargo && cargo version && clang --version && openssl version
     - run: scripts/reset-submodule.cmd
-    - run: cargo test --features "openssl" --all
     - run: cargo test --features "openssl-vendored" --all
+    - run: cargo test --features "openssl" --all
```

Comment:
On my local machine, sometimes the result of `openssl` compilation will affect `openssl-vendored`, such as the path information of find_package. I feel puzzled about this. In order to avoid possible problems, I adjusted the order to ensure that openssl-vendored is indeed compiled.

Reply:
What if "openssl-vendored" changes the behavior of "openssl"?

Reply:
All I can think of is `cargo clean` 😆